examples/gke-a3-ultragpu/gke-a3-ultragpu.yaml

```diff
@@ -53,21 +53,22 @@ deployment_groups:
 - group: primary
   modules:
   - id: gke-a3-ultra-net-0
-    source: github.com/GoogleCloudPlatform/cluster-toolkit.git//modules/network/vpc?ref=e0c690b
+    source: modules/network/vpc
     settings:
-      network_name: gke-a3-ultra-net-0
+      network_name: $(vars.deployment_name)-net-0
```

Comment:
Network name would be limited to 63 characters, do we have a limitation on the len(vars.deployment_name) ?

Reply:
Also update the name of the PR to say something like "Update A3U blueprint to remove commit refs and remove hardcoded network names"

Reply:
Let me check on the limitation on len(vars.deployment_name)
Will update the PR name